OWML.ModLoader/Owo.cs

```diff
@@ -112,5 +113,17 @@ private IModBehaviour InitializeMod(Type modType, IModHelper helper)
             }
         }
 
+        private static IModConsole GetConsole(IOwmlConfig owmlConfig, IModLogger logger, IModManifest owmlManifest)
```

Comment:
Could you use functionality from the Output class instead?

Reply:
Good catch, forgot to remove that one